.travis.yml

```diff
@@ -21,8 +21,8 @@ script:
   - tox
 env:
   global:
-    - DJANGO_SETTINGS_MODULE=tests.settings
-    - DATABASE_URL='postgres://postgres@localhost:5432/saleor'
+    - DATABASE_URL="postgres://postgres@localhost:5432/saleor"
+    - SECRET_KEY="irrelevant"
```

Comment:
Why are we setting the `SECRET_KEY` environment variable? It's already defined here:
https://github.com/mirumee/saleor/blob/cd74214e5fa916219517bc52ad4704754da5f553/tests/settings.py#L6

Reply:
Right. I removed the ```SECRET_KEY `` in circle CI.

But the one in travis is kept since ```collectstatic``` in tox requires it.